app/javascript/components/program/Section.js

```diff
@@ -0,0 +1,26 @@
+import React, { PureComponent } from "react";
+import { Link } from 'react-router-dom'
+
+class Section extends PureComponent {
```

Comment:
This will evolve to a more complex page showing a section of the program. It will will have headings, paragraphs, bullets, etc. Does it still make sense to be a functional component?